backend/index.js

```diff
@@ -33,11 +33,6 @@ const influencerAuthenticationMiddleware = require("./middleware/influencers/inf
 const influencerLoggingMiddleware = require("./middleware/influencers/influencerLoggingMiddleware");
 const { getSocialData } = require("./controllers/influencerController");
 
-// Middleware for Influencers
-app.use("/influencers", influencerAuthenticationMiddleware); // Authentication for influencers
```

Comment:
I removed from here and added in the influencers router page. Because if they are in index file authentication will apply for signup and login it leads to unable to login the user.